MotDuJour/App/Sources/AppActions.swift

```diff
@@ -8,12 +11,37 @@ enum AppAction {
     case word(WordAction)
 }
 
-enum RecentsAction {
-    case didTapWord
-    case didTapClearAll
-}
+extension AppAction {
+    static let homePrism: Optics.Prism<AppAction, HomeAction> = Optics.Prism<AppAction, HomeAction>(
+        extract: { appAction in
+            if case let .home(action) = appAction {
+                return action
+            } else {
+                return nil
+            }
+        },
+        embed: { AppAction.home($0) }
+    )
+
+    static let favoritesPrism = Optics.Prism<AppAction, FavoritesAction>(
```

Comment:
Is this different from the CasePath for `AppAction.favorites`?

Reply:
no, but we don't have an API that supports the prism being a CasePath and the lens being a `Optics.Lens`; either the lens and prisms are both case/keypaths or both `Optics`